lib/helpers/tsconfig.json

```diff
@@ -0,0 +1,24 @@
hulz (Charles M. Schulz's son and grandson, respectively), this movie is as heartwarming as it is innovative and interesting - a perfect fit for our Royal kids matinees!Our favourite underdog Charlie Brown has a crush on his new neighbour, the Little Red-Haired Girl, while his pup has fallen into the depths of his fantasy world as a World War I flying ace, pursuing his nemesis the Red Baron, all while trying to win the interests of a beautiful poodle named Fifi. Charlie Brown takes Lucy's advice and embarks on a series of new activities to gain the interest of the Little Red-Haired Girl, but with his ever present lack of confidence, has trouble making an impression on her, but he keeps trying anyway (we've all been there, Charlie Brown). This adorable flick about young puppy love is a surefire way to thaw out from the January weather, and with state of the art 3-D animation, it is undoubtedly the Peanuts like you've never seen them before! ","url":"https://www.universe.com/events/the-peanuts-movie-tickets-toronto-V8RJNB?ref=ticketmaster","locale":"en-us","images":[{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_ARTIST_PAGE_3_2.jpg","width":305,"height":203,"fallback":true},{"ratio":"4_3","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_CUSTOM.jpg","width":305,"height":225,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_16_9.jpg","width":640,"height":360,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_PORTRAIT_3_2.jpg","width":640,"height":427,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RECOMENDATION_16_9.jpg","width":100,"height":56,"fallback":true},{"ratio":"3_2","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_3_2.jpg","width":1024,"height":683,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_RETINA_LANDSCAPE_16_9.jpg","width":1136,"height":639,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_16_9.jpg","width":1024,"height":576,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_TABLET_LANDSCAPE_LARGE_16_9.jpg","width":2048,"height":1152,"fallback":true},{"ratio":"16_9","url":"http://s1.ticketm.net/dam/c/8cf/a6653880-7899-4f67-8067-1f95f4d158cf_124761_EVENT_DETAIL_PAGE_16_9.jpg","width":205,"height":115,"fallback":true}],"sales":{"public":{"startDateTime":"2016-01-14T20:20:36Z","startTBD":false,"endDateTime":"2016-01-17T21:00:00Z"}},"dates":{"access":{"startDateTime":"2016-01-14T20:20:36Z","startApproximate":false,"endDateTime":"2016-01-17T21:00:00Z","endApproximate":true},"s
```

Comment:
Is this file necessary?  I don't see references to `tsconfig` anywhere in the PR.